src/components/Deals/Deals.js

```diff
@@ -1,10 +1,29 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+import axios from 'axios';
+import SearchBar from '../Home/SearchBar';
 import ItemsGrid from '../Home/ItemsGrid';
 
 function Deals() {
+
+    const [isLoaded, setIsLoaded] = useState(false)
+    const [items, setItems] = useState([]);
+
+    useEffect(() => {
+        axios('https://gp-super-store-api.herokuapp.com/item/list?sortDir=asc&isOnSale=true')
+        .then(res => {
+            setItems(res.data.items);
+            setIsLoaded(true);
+        })
+        .catch(err => console.log(err))
+    }, [])
+
     return (
         <div className="text-center">
-            <ItemsGrid showOnSale={true} />
+
+            <SearchBar />
+
+            { isLoaded ? '' : <h3>Loading...</h3> }
```

Comment:
you can change this line to `{ !isLoaded && <h3>Loading...</h3> }` instead of the ternary if you want. Both ways are fine though, I just think using `&&` is cleaner if you aren't rendering anything in one of the results of the ternary.